### PR TITLE
create memgraph-lab chart

### PR DESCRIPTION
**Pull request by tewnut**
- Supported templates: deployment, service, serviceaccount, ingress and hpa.
- Tested with k8s 1.27, memgraph 2.16.

### CODE REVIEW

File: charts/memgraph-lab/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+To install the Memgraph Lab Helm Chart, follow the steps below:
+```
+helm install <release-name> memgraph/memgraph-lab
+```
+Replace `<release-name>` with a name of your choice for the release.
+
+## Changing the default chart values
+To change the default chart values, run the command with the specified set of flags:
+```
+helm install <resource-name> memgraph/memgraph --set <flag1>=<value1>,<flag2>=<value2>,...
```
Comment (antejavor) [inline charts/memgraph-lab/README.md:14]: This should probably be `memgraph/memgraph-lab` ?

**Review by antejavor — CHANGES_REQUESTED** [review]
I think this looks great, just two small comments. Good job 🚀 

Later we can maybe introduce the Memgraph dependency, so the Memgraph-Lab becomes the both Memgraph and Lab in same deployment. But we will probably do that part. 

Can you just update the available charts  in this PR: https://github.com/memgraph/helm-charts?tab=readme-ov-file#available-charts?

**Comment by tewnut** [comment]
Yes, certainly I will update the readme very soon.

Stacking all components into one chart is a popular design. At the same time keeping them separate (and highly independent of each other) has the beauty of its own. In my opinion, separating them is more beneficial, it may encourage community to further develop other UI (among other) components, which will boost adoption of Memgraph as a whole.

File: charts/memgraph-lab/README.md (hunk 1 above)
Comment (tewnut) [reply charts/memgraph-lab/README.md]: yep, updated in latest commit.

**Review by antejavor — APPROVED** [review]
Looks good 🚀

**Pull request merged**